### DISCUSSION TITLE
pacman -Syu Problem

### DISCUSSION THREAD

**Discussion by teslasparks301** [rern/rAudio#725] [Bug Reports]
First, I would like to say that rAudio-1 is the best rpi music player I have ever used. It is quite responsive and I have quite a large library. It works consistently, a problem I had with other apps. And the snapserver works great. 

Now my problem. According to my rAudio-1 update page, I need to install the latest image. I am using rAudio-1-64bit-20211203.img. It boots just fine. Then I run pacman -Syu and reboot. The rpi, a 3B+, is DOA. I have done this twice. There is no log. The console never displays anything. It is just dead. My understanding is that this image version has the pacman fix. I have nothing in /boot anyways. It is empty. I need to compile the driver for my wifi dongle and cannot find the headers for the kernel version used in the image so I need to upgrade to get the newer kernel and the headers for it.

Is there still a pacman problem? Is there a work-around? Am I doing something stupid again?
Please advise.

**Comment by R101G** [comment]
@teslasparks301 

If there is nothing in the boot partition, it certainly won't boot, but I don't know why it would be empty.

If you get past that problem, remember that you probably need the Pi 4 headers for 64 bit, even though it is a Pi 3.

**Reply by teslasparks301** [reply]
There is a bit of a misunderstanding. I was referring to the boot folder in the ROOT partition. There are many files in the BOOT partition.

I gather from your reply that pacman should be working OK. Is that so?

**Reply by klassikmann** [reply]
@teslasparks301 
@R101-Github 
Please have a look into the following threads:
[https://archlinuxarm.org/forum/viewtopic.php?f=15&t=15759](url)
[https://github.com/raspberrypi/firmware/issues/1671](url)

Solution:
Before you reboot insert "kernel=kernel8.img" into /boot/config.txt
If your system is dead already, edit the SD-card at your PC. The filesystem type /boot is formatted as fat32, so you are able to edit the config.txt even under Windows.

**Reply by ceejayemm** [reply]
@teslasparks301 

Also have a look at #620 

@klassikmann - the links in your reply above don't work for me, I get a 404 error on each link

Chris

**Reply by klassikmann** [reply]
@ceejayemm 
Seems to be a problem in the URL expansion (or I don't know how to insert links in the forum). Copy the link contents in new browser tabs.

**Reply by teslasparks301** [reply]
@klassikmann There is much joy! That addition to the config worked. I would have never have figured that out.
Many Thanks.

**Reply by R101G** [reply]
@teslasparks301 Sorry, I completely misunderstood the situation. I use a 32 bit image with my Pi 3, but a 64 bit image had the same problem and the same solution. So thanks to @klassikmann from me too.

**Reply by klassikmann** [reply]
The bug has been fixed upstream at Tue Jan 4 20:40:52 2022 UTC with raspberrypi-bootloader 20220104-1. "pacman -Syu" is safe now. No need anymore to edit /boot/config.txt.
